Velaptor/NativeInterop/OpenGL/IGLInvoker.cs

```diff
@@ -591,4 +581,15 @@ internal interface IGLInvoker : IDisposable
     /// <param name="pixels">[length: COMPSIZE(format,type,width,height)] Specifies a pointer to the image data in memory.</param>
     void TexImage2D<T>(GLTextureTarget target, int level, GLInternalFormat internalformat, uint width, uint height, int border, GLPixelFormat format, GLPixelType type, byte[] pixels)
         where T : unmanaged;
+
+    /// <summary>
```

Comment:
@KinsonDigitalAdmin Here, I don't know what to write as a documentation.

Reply:
Thanks for the heads up.

Here, just use `/// <inheritdoc cref="GL.DebugMessageCallback"/>`.  Even though there are no docs added to that by the SILK team, once they do add docs one day, it will just be automatic and that is fine with me.  

Thanks!! 😀